app/views/users/sessions/new.html.erb

```diff
@@ -37,3 +60,20 @@
 <%= link_to user_google_oauth2_omniauth_authorize_path, method: :post do %>
   <img src="../img/btn_google_signin_dark_focus_web@2x.png" alt="Sign in with Google">
 <% end %>
+
+<script>
+    document.getElementById('toggle_password').onclick = function () {
```

Comment:
Can we package this into a Stimulus controller? That's the way we want to encourage sprinkling JavaScript into our pages. You can see an example here: https://github.com/rubyforgood/human-essentials/blob/8839e19eb67ed92fb4c96fa7a98beab2ddf9e09f/app/javascript/controllers/highchart_controller.js#L32-L32 and it's used https://github.com/rubyforgood/human-essentials/blob/8839e19eb67ed92fb4c96fa7a98beab2ddf9e09f/app/views/shared/_highcharts.html.erb#L4-L4

Reply:
Hi Dorner,
I've refactored the password toggle into a Stimulus controller as requested. It’s now encapsulated neatly in the password_visibility_controller.js, making our JS integration more structured.